lib/pages/page_settings.dart

```diff
@@ -21,6 +23,7 @@ class SettingsPage extends StatefulWidget {
 }
 
 class _SettingsPageState extends State<SettingsPage> {
+  final GlobalKey<PlaylistSelectorState> _playlistSelectorKey = GlobalKey();
```

Comment:
Kann mir jemand das GlobalKey Konzept hier erklären?

Reply:
GlobalKeys werden dafür verwendet um auf Methoden von anderen StatefulWidgets zuzugreifen. Hiermit kann ich eine Methode in dem playlistSelector Widget aufrufen um den State von diesem von einem anderen Widget aus zu manipulieren.